test/ProgressOnderwijsUtils.Tests/EnumerableExtensionsTests.cs

```diff
@@ -56,7 +56,7 @@ public void TestFirstIndexOfDups()
     [Fact]
     public void WhereNotNull_RemovesNullsWithoutCompilerWarningForReferenceTypes()
     {
-        IEnumerable<string?> sampleNullableData = new[] { "test", null, "this", };
+        var sampleNullableData = new[] { "test", null, "this", }.AsEnumerable();
```

Comment:
Don't they have fancy syntax nowadays?

```suggestion
        var sampleNullableData = ["test", null, "this"].AsEnumerable();
```

Reply:
The collection expression syntax is sort of like the expression-tree-vs-delegate distinction in that it's target typed.  It doesn't have it's own type, which IMNSHO is a bad design. Unfortunately that does mean that collection expressions do not support extension method resolution at all right now, and also currently cannot be stored in variables; they _need_ a target type.

Reply:
🤷

Reply:
Inderdaad :-D